internal/drivers/manager.go

```diff
@@ -558,6 +558,7 @@ func (m *Manager) StartInstance(ctx context.Context, poolName, instanceID string
 		return nil, fmt.Errorf("start_instance: pool name %q not found", poolName)
 	}
 
+	logrus.WithField("instanceID", instanceID).Infoln("Starting vm from hibernate state")
```

Comment:
VM might not be hibernated here, we should move this log message after the hibernate check